inst/components/scss/accordion.scss

```diff
@@ -1,18 +1,19 @@
 .accordion {
   .accordion-header {
-    @extend .h2;
+    @include bootstrap-heading($h2-font-size);
```

Comment:
Why did we need this change? We talked about this in person, but the details didn't quite sink in.

Reply:
Because we can't modify the Bootstrap bundle

```R
> sass::sass(".foo {color: red}; .bar{ @extend .foo; }")
/* CSS */
.foo, .bar {
  color: red;
}
```